### PR TITLE
{bio}[gompi/2020b] MaxBin v2.2.7

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3501.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/2a5d6830ffbe03640499f021d7082a04 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3101.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/0a15eccab402edc50badb03168f87a41 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 2 out of 3 (3 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/98ceaa923aa7626394c964908e377fa0 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 2 out of 3 (3 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.2
See https://gist.github.com/38dd598b0e5c8419add6279bf5b3a172 for a full test report.

**Comment by verdurin** [comment]
@boegel checksum failure reported here.

**Comment by Micket** [comment]
Test report by @Micket
**FAILED**
Build succeeded for 4 out of 5 (3 easyconfigs in total)
vera-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/02c0db37d97f91ac69c5757f767cfd60 for a full test report.

**Comment by boegel** [comment]
> @boegel checksum failure reported here.

Indeed, fixed in c8a9525, see also #12869.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/f/FragGeneScan/FragGeneScan-1.31-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%(name)s%(version)s.tar.gz']
+checksums = ['cd3212d0f148218eb3b17d24fcd1fc897fb9fee9b2c902682edde29f895f426c']
+
+builddependencies = [('binutils', '2.35')]
+
+dependencies = [("Perl", "5.32.0")]
+
+fix_perl_shebang_for = ['*.pl']
+
```
Comment (branfosj) [inline easybuild/easyconfigs/f/FragGeneScan/FragGeneScan-1.31-GCCcore-10.2.0.eb:20]:
```suggestion

prebuildopts = "make clean && "
```

The fix in #12870 should also be applied here.

**Review by branfosj — CHANGES_REQUESTED** [review]

**Review by branfosj — APPROVED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ generoso
CORE_CNT=16

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/b80b786f811b188bb5c6d1993409d9f4 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**